src/node_api_types.h

```diff
@@ -68,6 +75,9 @@ typedef enum {
   napi_name_expected,
   napi_function_expected,
   napi_number_expected,
+#ifdef NAPI_EXPERIMENTAL
+  napi_bigint_expected,
+#endif  // NAPI_EXPERIMENTAL
```

Comment:
We should only ever add enum values to the bottom since they need to be stable over time.